eos/effects/fighterabilityevasivemaneuvers.py

```diff
@@ -0,0 +1,24 @@
+# Not used by any item
+"""
+Since fighter abilities do not have any sort of item entity in the EVE database, we must derive the abilities from the
+effects, and thus this effect file contains some custom information useful only to fighters.
+"""
+# User-friendly name for the ability
+displayName = "Evasive Maneuvers"
```

Comment:
What he said. It's my hacky way of implementing it before they added this data to the cache:

https://gist.github.com/blitzmann/935c2d5b178d62205ba88a30463e8eda

Which I don't believe is even in the SDE or on CREST. v0v